app/javascript/angular/code/controllers/_mobile_sessions_map_ctrl.js

```diff
@@ -57,8 +57,6 @@ export const MobileSessionsMapCtrl = (
       location: "",
       tags: "",
       usernames: "",
-      gridResolution: 25,
-      crowdMap: false,
```

Comment:
Changing these values doesn't change the behavior of the website, so I think they're obsolete. They are set in `elm.js`.